spec/models/form/sales/questions/mortgage_amount_spec.rb

```diff
@@ -46,4 +46,20 @@
   it "has correct min" do
     expect(question.min).to be(1)
   end
+
+  context "when the mortgage is not used" do
+    let(:log) { create(:sales_log, :completed, mortgageused: 2, deposit: nil) }
+
+    it "is marked as derived" do
+      expect(question.derived?(log)).to be true
+    end
+  end
+
+  context "when the mortgage is used" do
+    let(:log) { create(:sales_log, :completed, mortgageused: 1) }
+
+    it "is marked as derived" do
+      expect(question.derived?(log)).to be false
+    end
+  end
```

Comment:
is there a reason these assertions don't follow the same format as the one above you changed (which looks really nicely readable), (i.e. expect(q).to be_derived)